test-commons/src/main/java/no/unit/nva/publication/PublicationGenerator.java

```diff
@@ -234,14 +233,22 @@ private static FileSet sampleFileSet() {
 
     private static License randomLicense() {
         return new License.Builder()
-            .withIdentifier(randomString())
+            .withIdentifier(randomWord())
             .withLink(randomUri())
             .withLabels(randomMap())
             .build();
     }
 
+    private static String randomWord() {
+        return FAKER.lorem().word();
+    }
+
     private static Map<String, String> randomMap() {
-        return Map.of(randomString(), randomString());
+        return Map.of(randomLanguage(), randomString());
+    }
+
+    private static String randomLanguage() {
+        return NORWEGIAN_LANGUAGE_KEY;
```

Comment:
Still not very random, I'd suggest one of ["en", "nb", "nn"] as a small selection

Reply:
Randomized